controllers/auth.js

```diff
@@ -106,7 +106,7 @@ exports.login = (req, res, next) => {
         );
         res.status(200).json({
             token: token,
-            userId: loadedUser._id.toString()
+            userId: loadedUser.name.toString()
```

Comment:
Can you change this to ID again? We need User ID to get user information and track user's activity in the session.

Reply:
ah I see, i will re add that, but we also need to send name, that's what we will show, something like Hi Juyee on the top right when u login